src/Elm2Nix.hs

```diff
@@ -51,16 +53,21 @@ parseElmJsonDeps obj =
   case obj of
     Object hm -> do
       deps <- tryLookup hm "dependencies"
-      case deps of
-        Object dhm -> do
-          direct   <- tryLookup dhm "direct"
-          indirect <- tryLookup dhm "indirect"
-          liftM2 (++) (parseDeps direct) (parseDeps indirect)
-        v -> Left (UnexpectedValue v)
+
+      let libstyleDeps = parseDeps deps
+      if Either.isRight libstyleDeps then
+        libstyleDeps
```

Comment:
to my surprise, I don't know about a better way to do this. But I still feel there might be one. Maybe to use `either` function?

Reply:
It's fine :)